### ISSUE TITLE
Test if basic NN works for images

### ISSUE THREAD

**Issue by caicai67** [caicai67/DogEmotionRecognition#34]
- [ ] add labels to training data
- [ ] Convert images to array
- [ ] Feed training data into NN
- [ ] See if test data is classified correctly
- [ ] Evaluate the neural net based on
    - [ ] time of results
    - [ ] space used by NN
    - [ ] accuracy
